src/ci/bin/test_slurm.inc.sh

```diff
@@ -21,21 +21,21 @@ cromwell::build::slurm::setup_slurm_environment() {
     # https://slurm.schedmd.com/
     sudo apt-get update
 
-    # Try the Lawrence Livermore National Laboratory (LLNL) version first
-    sudo apt-get install -y slurm-llnl || sudo apt-get install -y slurm-wlm
+    sudo apt-get install -y slurm-wlm
+    #As an alternative to 'slurm-wlm', you may also try 'slurm-llnl'.
+    #If you use 'slurm-llnl', change the below tee path to '/etc/slurm-llnl/slurm.conf'
+    #For reasons that are unclear, slurm-llnl doesn't work in Github Actions, and slurm-wlm doesn't work in Travis.  
```

Comment:
Thanks for adding this comment! Might be useful to add a little more context explaining that we changed this when we moved off Travis to GHA, and point to a ticket.

Reply:
All set, good idea!